examples/ads.amp.html

```diff
@@ -265,6 +266,12 @@
       data-host="talos.adman.gr">
   </amp-ad>
 
+  <h2>AdmanMedia</h2>
+  <amp-ad width="300" height="250"
```

Comment:
Thanks for you valuable remarks that helped us improve the code quality. We have started the internal process for CORS header setting and hopefully in a few days we will be all set.